src/groupmod.c

```diff
@@ -580,52 +581,36 @@ static void prepare_failure_reports (void)
 #endif
 	info_passwd.action  = pw;
 
-	gr = stpeprintf(gr, gr_end,
-	                "group %s/%ju", group_name, (uintmax_t) group_id);
+	gr = stpeprintf(gr, gr_end, "group %s/%ju", group_name,
+	                                            (uintmax_t) group_id);
```

Comment:
Can you reduce the indentation so that this line starts in line with the parenthesis of the previous line?

Reply:
Yup.  Done.